src/core/Chip/styles.css

```diff
@@ -0,0 +1,86 @@
+.container {
```

Comment:
group and sort CSS based on https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Reply:
ok